src/test/java/com/cloudbees/jenkins/plugins/sshagent/SSHAgentBuildWrapperTest.java

```diff
@@ -106,6 +106,9 @@ public void sshAgentUnavailable() throws Exception {
 
         FreeStyleProject job = r.createFreeStyleProject();
 
+        SSHAgentBuildWrapper sshAgent = new SSHAgentBuildWrapper(credentialIds, false);
+        job.getBuildWrappersList().add(sshAgent);
+
```

Comment:
But I thought the point of _this_ test was to show that the `ssh` command _would_ fail if you did not add the wrapper?